### ISSUE TITLE
[Bug] benchmarking on the Puyu model for several minutes, the server became unavailable

### ISSUE THREAD

**Issue by zhulinJulia24** [InternLM/lmdeploy#611]
### Checklist

- [X] 1. I have searched related issues but cannot get the expected help.
- [X] 2. The bug has not been fixed in the latest version.

### Describe the bug

env info:
lmdeploy version: 0.0.12
model info: puyu-123b

during profile serving, an error occurred (`StatusCode.TRITON_SERVER_ERR, [StatusCode.UNAVAILABLE]` ), then the server became unavailable. 

 The error call stack was similar to Issue #325 , but the scenario and impact are different. I cannot benchmark puyu model due to this issue. 

### Reproduction

1. serving with docker container 
2. do profile serving by using 
`python benchmark/profile_serving.py 10.140.0.187:33337 /mnt/models-new/puyu-123bXXX/triton_models/tokenizer/tokenizer.model ../ShareGPT_V3_unfiltered_cleaned_split.json 48 `
3. during warmup, the benchmark profile_serving error occurs
4. then start client, input some prompt, docker error occurs
`python3 -m lmdeploy.serve.client 10.140.0.187:33337`

### Error traceback

```Shell
docker log:

[TM][INFO] [forward] Enqueue requests
[TM][INFO] [forward] Wait for requests to complete ...
[TM][WARNING] [verifyRequests] Skipping invalid infer request for id 1, code = 1
W1025 09:25:10.801643 1 libfastertransformer.cc:1123] response is nullptr


benchmark profile_serving output: 

[StatusCode.UNAVAILABLE] Socket closed
10/25 17:23:51 - service.ft - ERROR - /home/zhulin1/miniconda3/envs/lmdeploy0.0.12/lib/python3.10/site-packages/lmdeploy/serve/turbomind/chatbot.py - stream_consumer - 628 - got error from turbomind, code StatusCode.TRITON_SERVER_ERR, [StatusCode.UNAVAILABLE] Socket closed, token 1029
```

**Comment by lvhan028** [comment]
Please share the log of the server

**Comment by zhulinJulia24** [comment]
docker log is attached.

docker log:

[TM][INFO] [forward] Enqueue requests
[TM][INFO] [forward] Wait for requests to complete ...
[TM][WARNING] [verifyRequests] Skipping invalid infer request for id 1, code = 1
W1025 09:25:10.801643 1 libfastertransformer.cc:1123] response is nullptr


benchmark profile_serving output: 

[StatusCode.UNAVAILABLE] Socket closed
10/25 17:23:51 - service.ft - ERROR - /home/zhulin1/miniconda3/envs/lmdeploy0.0.12/lib/python3.10/site-packages/lmdeploy/serve/turbomind/chatbot.py - stream_consumer - 628 - got error from turbomind, code StatusCode.TRITON_SERVER_ERR, [StatusCode.UNAVAILABLE] Socket closed, token 1029

**Comment by zhulinJulia24** [comment]
The docker log can be consistently reproduced, but docker error can not be reproduced consistently.

**Comment by lvhan028** [comment]
@AllentDan Please help check this issue

**Comment by AllentDan** [comment]
Did you turn the proxy off? Did it happen only for the above model?

**Comment by zhulinJulia24** [comment]
I didn't turn of the proxy. But the issue only happen on newest puyu model and lmdeply.

**Comment by AllentDan** [comment]
This was solved by turning the proxy off and using `0.0.0.0:33337` URL. If there are any new updates, you may reopen the issue or open a new one.
